_rules/SC3-1-2-lang-valid.md

```diff
@@ -78,7 +82,7 @@ The `lang` attribute specified has a non-empty value & a valid primary language
 
 #### Passed example 4
 
-The `lang` and `xml:lang` attribute specified has a non-empty value & a valid primary language subtag.
+The `lang` and `xml:lang` attributes values that are not the empty string ("") and has valid primary language subtags.
```

Comment:
The `lang` and `xml:lang` attributes values are not the empty string ("") and have valid primary language subtags.